.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,40 @@
+name: Bug Report
+description: File a bug report for the ui-kit
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a bug report, please search for the behaviour in the existing issues. 
+        
+        ---
+        
+        Thank you for taking the time to file a bug report. To address this bug as fast as possible, we need some information.
+  - type: input
+    id: version
+    attributes:
+      label: ui-kit version
+      description: Which UI-kit version are you using?
```

Comment:
And then the version would need to match one of the above product. `ui-kit` does not have a version, it's only the repo.